content/post/uncoordinated-omission/index.adoc

```diff
@@ -0,0 +1,476 @@
+
+### How to use Bpf Validator
+
+1. Build  `bpf_validator` repository from https://github.com/johnaohara/bpf_validator[github repository], following the https://github.com/johnaohara/bpf_validator/blob/main/README.md#prerequisites[build instructions]
+
+2. Run `bpf_validator` tool to start capturing HTTP Round Trip Times (RTT)
+
+[source,sh,indent=8]
+----
+$ sudo ./bpf_validator
+...
+libbpf: map 'bpf_vali.rodata': created successfully, fd=7
+libbpf: map '.rodata.str1.1': created successfully, fd=8
+
+bpf_validator is monitoring port: 8000
+----
+
+[start=3]
+3. Send `SIGINT` (CtRL+C) or `SIGTERM` to the running bpf_validator process to stop monitoring and print the summary statistics
+
+
+[source,sh,indent=8]
+----
+$ sudo ./bpf_validator
+...
+bpf_validator is monitoring port: 8000
+^C
+
+Printing HdrHistogram stats:
+
+    Value   Percentile   TotalCount 1/(1-Percentile)
+
+    0.000     1.000000            0          inf
+#[Mean    =         -nan, StdDeviation   =         -nan]    <1>
+#[Max     =        0.000, Total count    =            0]    <2>
+#[Buckets =           22, SubBuckets     =         2048]    <3>
+
+50.0th Percentile: 0.000000                                 <4>
+90.0th Percentile: 0.000000
+95.0th Percentile: 0.000000
+99.0th Percentile: 0.000000
+99.9th Percentile: 0.000000
+99.99th Percentile: 0.000000
+
+Detailed Percentile spectrum:
+    Value   Percentile   TotalCount 1/(1-Percentile)
+
+    0.000     1.000000            0          inf
+#[Mean    =         -nan, StdDeviation   =         -nan]
+#[Max     =        0.000, Total count    =            0]
+#[Buckets =           22, SubBuckets     =         2048]
+
+0 requests in 0.000000s                                     <5>
+Av Throughput: -nan req/sec                                 <6>
+----
+<1> Summary Statistics
+<2> Total number of samples in histogram
+<3> Histogram bucketing details
+<4> Percentile response Times
+<5> Total request count and time between first and last HTTP packet
+<6> Average throughput: total requests / time difference between first and last HTTP packet
+
+## Lets Test!
+
+For a simple test, we can run a https://quarkus.io/[Quarkus] application which you can obtain by clicking "Generate your application" at https://code.quarkus.io/
+
+[source,sh,indent=4]
+----
+
+$ mvn clean package
+...
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+...
+
+$ mvn quarkus:dev -Dquarkus.http.port=8000
+...
+__  ____  __  _____   ___  __ ____  ______ 
+--/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
+-/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
+--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
+2024-12-03 14:59:37,947 INFO  [io.quarkus] (Quarkus Main Thread) code-with-quarkus 1.0.0-SNAPSHOT on JVM (powered by Quarkus 3.17.2) started in 2.072s. Listening on: http://localhost:8000
```

Comment:
2 seconds is a lot - this is running in dev mode for some reason?
I mean, TBH is not important, just curiousity